src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -927,6 +1444,107 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		037C3CAD2991A44B00B7EEE2 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CHIP_BUILD_TOOLS = true;
+				CHIP_ROOT = "$(PROJECT_DIR)/../../..";
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CODE_SIGN_IDENTITY = "-";
+				CODE_SIGN_STYLE = Manual;
```

Comment:
The other bits in this file have:
```
                               CODE_SIGN_IDENTITY = "Apple Development";
                               CODE_SIGN_STYLE = Automatic;
```

Which one is right and why?

Reply:
I think both are correct, because the Matter framework is a framework and does't "run". The darwin-framework-tool runs and needs to be signed to run. To sign it without a team, we need to configure sign to run locally. 

Here is the config with "Apple Development":
<img width="457" alt="Screenshot 2023-02-08 at 10 02 39 AM" src="https://user-images.githubusercontent.com/12054330/217614240-cc621e57-2148-4824-828a-29675ca5e559.png">

Here is the current config:
<img width="479" alt="Screenshot 2023-02-08 at 10 04 43 AM" src="https://user-images.githubusercontent.com/12054330/217614700-c85cfcd4-760c-431a-9749-50c56cda9647.png">

Reply:
I was able to make Automatic.